sql/api/src/main/scala/org/apache/spark/sql/streaming/StatefulProcessor.scala

```diff
@@ -107,6 +107,8 @@ trait StatefulProcessorWithInitialState[K, I, O, S] extends StatefulProcessor[K,
    *
```

Comment:
In the line above - could we please update to include package private for sql ?

`private[sql] trait StatefulProcessorWithInitialState` ?